### ISSUE TITLE
How/when will CSS6 be defined?

### ISSUE THREAD

**Issue by romainmenke** [CSS-Next/css-next#106]
Currently we are mid 2024. 

There is a `CSS5` category for properties and features introduced in a spec around 2019 - 2024.
There is also a `Next` placeholder for future work.

What happens Januari 1st 2025?

Is a new `CSS6` category created where all `2025-2029` works goes into?

Or does all the `2025-2029` work remain in `Next` until the end of `2029` to only then move everything over to `CSS6`?

In other words, is the latest numbered category a living document?
Or is `Next` the only living document?

-----

I don't have a strong opinion either way, but I didn't see any mention of this, maybe I missed it :)
